test/unit/specs/server/live-loan-fetch.spec.js

```diff
@@ -74,6 +74,17 @@ describe('live-loan-fetch', () => {
 			await testSortParsing('1234', null);
 		});
 
+		it('have flss call defined with userId', async () => {
+			fetch.mockClear();
+			fetch.mockResolvedValue({ json: () => {} });
+
+			await fetchLoansByType('user', '1234', true);
+
+			expect(fetch).toBeDefined();
+			const { variables } = JSON.parse(fetch.mock.calls[0][1].body);
+			expect(variables.userId).toEqual(1234);
```

Comment:
Can we also ensure that the FLSS method is called (or a query associated with the method)?